internal/util/csiconfig.go

```diff
@@ -58,19 +58,19 @@ type clusterInfo struct {
 // 	},
 // 	...
 // ]
-func readClusterInfo(pathToConfig, clusterID string) (clusterInfo, error) {
-	var config []clusterInfo
+func readClusterInfo(pathToConfig, clusterID string) (ClusterInfo, error) {
+	var config []ClusterInfo
 
 	// #nosec
 	content, err := ioutil.ReadFile(pathToConfig)
 	if err != nil {
 		err = fmt.Errorf("error fetching configuration for cluster ID (%s). (%s)", clusterID, err)
-		return clusterInfo{}, err
+		return ClusterInfo{}, err
```

Comment:
I prefer `nil` returns when an error happens. It was an empty structure before, but that is a little more sensitive to problems as it can be used without detecting an error.

Reply:
Thanks, I'll make the required changes.